packages/test/stochastic-test-utils/api-report/stochastic-test-utils.alpha.api.md

```diff
@@ -6,10 +6,6 @@
 
 /// <reference types="mocha" />
 
-export { AsyncGenerator_2 as AsyncGenerator }
-
-export { Generator_2 as Generator }
```

Comment:
Similar scenario to comment above, these types are `@internal` so not sure what they were doing here.

Reply:
yeah. this package doesn't even need API reports really anyway.

Reply:
This was a bug fixed in https://github.com/microsoft/rushstack/pull/5211